### ISSUE TITLE
[Number precision] Precision lost rendering large integers for NGSIv2 attribute values

### ISSUE THREAD

**Issue by fgalan** [telefonicaid/fiware-orion#2506]
The following behaviour has been detected, related with large integer values in NGSIv2 attributes:

```
$ curl -vvvv localhost:1026/v2/entities -H 'Content-Type: application/json' -d @- <<EOF
{
  "id": "E",
  "type": "T",
  "A": {
    "value": 2523232323232323,
    "type": "Number"
  }
}
EOF

* Hostname was NOT found in DNS cache
*   Trying ::1...
* Connected to localhost (::1) port 1026 (#0)
> POST /v2/entities HTTP/1.1
> User-Agent: curl/7.38.0
> Host: localhost:1026
> Accept: */*
> Content-Type: application/json
> Content-Length: 89
>
* upload completely sent off: 89 out of 89 bytes
< HTTP/1.1 201 Created
< Connection: Keep-Alive
< Content-Length: 0
< Location: /v2/entities/E?type=T
< Fiware-Correlator: ea1f4d1c-78f9-11e6-b398-d48564c29d20
< Date: Mon, 12 Sep 2016 15:02:25 GMT
<
* Connection #0 to host localhost left intact

$ curl localhost:1026/v2/entities/E
{"id":"E","type":"T","A":{"type":"Number","value":2.52323e+15,"metadata":{}}}
```

Thus, the render value is not exactly the one it was introduced. However, in DB the stored value is correct:

```
> db.entities.find().pretty()
{
        "_id" : {
                "id" : "E",
                "type" : "T",
                "servicePath" : "/"
        },
        "attrNames" : [
                "A"
        ],
        "attrs" : {
                "A" : {
                        "type" : "Number",
                        "creDate" : 1473692545,
                        "modDate" : 1473692545,
                        "value" : 2523232323232323
                }
        },
        "creDate" : 1473692545,
        "modDate" : 1473692545
}
```

Not sure if this is a desired behaviour or not and how difficult would be to change it in the render logic. By the moment we create this issue to be studied in the future.

**Comment by fgalan** [comment]
This issue is not especifically related with NGISv2, so APIv2 label is removed. The issue keeps opened.

CC: @jmcanterafonseca

**Comment by fgalan** [comment]
Fixed by PR #3229 (in 1.15.0) but ftest re-enabled later in PR #3294.
